docs/server/commands.rst

```diff
@@ -324,6 +324,9 @@ Available options:
 
 .. django-admin-option:: --from-revision
 
+  .. versionchanged:: 2.8.0 You cannot specify this option and
```

Comment:
`--since` being a new option, not sure if it's worth adding extra noise with `versionadded` here. I'd personally remove the directive and leave it as a normal paragraph.